src/components/AppShell.tsx

```diff
@@ -0,0 +1,26 @@
+import { AppShell, Navbar, Header } from '@mantine/core';
+import NewsComponent from './NewsComponent';
+import TodoList from './TodoList';
+import WeatherComponent from './WeatherComponent';
+
+function Shell() {
+  return (
+    <AppShell
+      padding="md"
+      navbar={
+        <Navbar width={{ base: 300 }} height={800} p="xs">
+          <TodoList />
+        </Navbar>
+      }
+      header={
+        <Header height={200} p="xs">
+          <WeatherComponent />
```

Comment:
Similar thing with Weather Component however We can fix this in the next ticket so happy for this to stay as is for now